Mathlib/LinearAlgebra/Matrix/BilinearForm.lean

```diff
@@ -127,7 +127,9 @@ theorem Matrix.toBilin'Aux_eq (M : Matrix n n R₂) : Matrix.toBilin'Aux M = Mat
 #align matrix.to_bilin'_aux_eq Matrix.toBilin'Aux_eq
 
 theorem Matrix.toBilin'_apply (M : Matrix n n R₂) (x y : n → R₂) :
-    Matrix.toBilin' M x y = ∑ i, ∑ j, x i * M i j * y j := Matrix.toLinearMap₂'_apply _ _ _
+    Matrix.toBilin' M x y = ∑ i, ∑ j, x i * M i j * y j :=
+  Finset.sum_congr rfl fun _  _ => Finset.sum_congr rfl fun _  _ => by
+          simp only [RingHom.id_apply, smul_eq_mul, mul_comm, mul_assoc]
```

Comment:
I would still like to use `toLinearMap₂'_apply` to unfold the definition, instead of doing so implicitly.
```suggestion
    Matrix.toBilin' M x y = ∑ i, ∑ j, x i * M i j * y j :=
  (Matrix.toLinearMap₂'_apply _ _ _).trans
    (by simp only [smul_eq_mul, mul_assoc, mul_comm, mul_left_comm])
```